VueJS/index.js

```diff
@@ -3,11 +3,37 @@ import ListItem from './components/ListItem.js'
 
 new Vue({
   el: '#app',
+  components: {
+    List,
+    ListItem
+  },
   data: {
-    todos: [
+    lastId: 3,
+    todos:  [
       { id: '1', message: 'Foo', },
       { id: '2', message: 'Bar', },
       { id: '3', message: 'Baz', }
     ]
-  }
+  },
+  methods: {
+    newTodo: function() {      
+      this.lastId++
```

Comment:
This is the 4th time I see this and it does not get better over time. :stuck_out_tongue: try to reduce state that needs to be updated. Remember:

> State is the root of all evil

What happens if you add items to the initial list of `todos`? Yes, you need to update this counter. This is a potential bug in the future.